File: sdkwork-ai-sdk-csharp/Models/Audio.cs
using System;
using System.Collections.Generic;
using System.Text.Json.Serialization;

namespace Ai.Models
{
    public class Audio
    {
        public string? Id { get; set; }
        public string? Data { get; set; }
        public string? Transcript { get; set; }
        public int? ExpiresAt { get; set; }
    }
}

File: sdkwork-ai-sdk-typescript/src/types/audio.ts
/** Audio response data */
export interface Audio {
  /** Unique identifier for this audio response */
  id?: string;
  /** Base64 encoded audio bytes */
  data?: string;
  /** Transcript of the audio */
  transcript?: string;
  /** The Unix timestamp (in seconds) for when this audio response was created */
  expires_at?: number;
}
